templates/test/options.test.js

```diff
@@ -6,11 +6,11 @@ describe('Options', () => {
       const config = {
         loader: {
           test: /\.js$/,
-          options: {boilerplate: true},
+          options: { boilerplate: true },
```

Comment:
nitpick 🐦 `name` (`options.name`) or `option` (`options.option`) may be more descriptive then `boilerplate`